Description: Change MAX_SENT_COUNT from 16 to 32 to resolve some cases easier.
Origin: upstream, http://unbound.nlnetlabs.nl/svn/ revision 3127
Bug-Ubuntu: https://bugs.launchpad.net/ubuntu/+source/unbound/+bug/1472510

Index: unbound-1.4.22/iterator/iterator.h
===================================================================
--- unbound-1.4.22.orig/iterator/iterator.h
+++ unbound-1.4.22/iterator/iterator.h
@@ -59,7 +59,7 @@ struct iter_priv;
 /** max number of referrals. Makes sure resolver does not run away */
 #define MAX_REFERRAL_COUNT	130
 /** max number of queries-sent-out.  Make sure large NS set does not loop */
-#define MAX_SENT_COUNT		16
+#define MAX_SENT_COUNT		32
 /** at what query-sent-count to stop target fetch policy */
 #define TARGET_FETCH_STOP	3
 /** how nice is a server without further information, in msec 
